transformation/src/interfaces/jobResult/jobEvent.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * This interface represents a Transformation Job that can be consumed from
+ * a channel.
+ * 
+ * It contains
+ * 
+ * @field pipelineID       pipeline ID of the request
+ * @field func             function to apply to the data (-> transformation) 
+ * @field data             data to be transformed
+ */
+export default interface JobEvent {
+    pipelineID: number      // pipeline ID of the request
+
+    func: string;           // function to apply to the data (-> transformation)
+
+    data: object;           // data to be transformed
```

Comment:
Why do we need that? What is a JobEvent? 

Is it the communication between adapter and transformation? 
Then we do have a DataImportFinishedEvent with `datasourceId` as member, maybe containing the `data` if modeled as fat event. 

If it's for communication between UI and transformation for the preview of transformations, then use the REST API.

Reply:
A jobEvent is the Event that is published by the adapter and consumed by the transformation service. 
We did not agreed yet wether to use a fat or a thin event for that, so I kept both possibilities in the Event definition.

Reply:
Let's use the thin event for now since there are fewer changes necessary. After merging the PR we can decide whether we want to use fat events and refactor if needed.